packages/dom/src/feature-app-container.ts

```diff
@@ -18,6 +18,19 @@ export interface DomFeatureApp {
   attachTo(container: Element): void;
 }
 
+/**
+ * Define a custom element named `feature-app-container` at the
```

Comment:
That reads like a challenge. Maybe use `Defines a ...`

Reply:
I'm pretty sure we use imperative in JSDoc everywhere (e.g. https://feature-hub.io/@feature-hub/core/classes/featureappmanager.html#getfeatureappscope)